Common/src/main/java/com/tabnineCommon/binary/fetch/BootstrapperSupport.java

```diff
@@ -67,6 +69,8 @@ private static Optional<BinaryVersion> downloadRemoteVersion(
 
   private static void notifyPluginInstalled() {
     if (ApplicationManager.getApplication() != null) {
+      PluginInstalled.Companion.setNewInstallation(true);
```

Comment:
I needed a new broadcaster to persist state between invocations as the objects are not all created at the same timing